ci-operator/step-registry/openshift/image-based/upgrade/recipient/presubmit/openshift-image-based-upgrade-recipient-presubmit-ref.yaml

```diff
@@ -0,0 +1,37 @@
+ref:
+  as: openshift-image-based-upgrade-recipient-presubmit
+  grace_period: 10m
+  commands: openshift-image-based-upgrade-recipient-presubmit-commands.sh
+  from_image:
+    name: release
+    namespace: openshift
+    tag: rhel-9-release-golang-1.21-openshift-4.16
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  credentials:
+  - name: ibu-backup-secret
+    namespace: test-credentials
+    mount_path: /var/run/ibu-backup-secret
+  - name: registry-pull-credentials
+    mount_path: /var/run/pull-secret
+    namespace: test-credentials
+  env:
+  - name: SEED_IMAGE
+    default: quay.io/openshift-kni/image-based-seeds
+    documentation: The repo where the seed image will be uploaded. The backup secret needs to have access to this repo
+  - name: SEED_VERSION
+    default: 4.14.10
+    documentation: The OCP version of the seed used for upgrade
+  - name: RECIPIENT_VERSION
+    default: 4.14.1
+    documentation: The OCP version that the recipient should start at
```

Comment:
These versions in the presubmit probably shouldnt be static set to 4.14 or there should be more jobs for 4.14/4.15/main right?

We can think of if we need to rename the job that is run or if we need to add more job configs I think

Reply:
I'm going to follow this PR up with an update to 4.15 but I'm avoiding more scope creep as this PR keeps getting larger and larger